torch/_dynamo/eval_frame.py

```diff
@@ -377,9 +378,14 @@ def get_compiler_config():
 
         @functools.wraps(fn)
         def _fn(*args, **kwargs):
+            any_arg_is_proxy = any(
+                isinstance(arg, torch.fx.Proxy)
+                for arg in itertools.chain(args, kwargs.values())
+            )
             if (
                 not isinstance(self, DisableContext)
                 and torch.fx._symbolic_trace.is_fx_tracing()
+                and any_arg_is_proxy
```

Comment:
What's going on here?

Reply:
is_fx_tracing flag is also set for make_fx. So we try to detect whether we're in symbolic_trace mode by checking if any arguments are wrapped as proxy.